src/App.css

```diff
@@ -36,3 +36,39 @@
     transform: rotate(360deg);
   }
 }
+.container{
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  height: 100%;
+  margin-top: 1%;
+}
+.mb-3 input{
+  width: 100%;
+  border-radius: 20px;
+  border: 1px solid #ccc;
+  padding: 10px;
+  margin-bottom: 10px;
+}
+.mb-3 input[text=text]{
+  border: 1px solid skyblue;
+}
+.container button{
```

Comment:
so this will style the buttons that are within a `container` class? why not just add a class right on the button?